package.json

```diff
@@ -115,12 +115,13 @@
     "wait-on": "^5.2.0"
   },
   "scripts": {
-    "start": "RADICLE_UPSTREAM_EXPERIMENTAL=false yarn start:app",
-    "start:experimental": "RADICLE_UPSTREAM_EXPERIMENTAL=true yarn start:app",
-    "start:app": "run-p --race rollup:watch proxy:start electron:start",
-    "start:release": "run-p --race rollup:watch proxy:start:release electron:start",
-    "start:experimental:release": "RADICLE_UPSTREAM_EXPERIMENTAL=true yarn start:release",
-    "start:test": "run-p --race rollup:watch proxy:start:test electron:start",
+    "start": "RADICLE_UPSTREAM_EXPERIMENTAL=false RADICLE_UPSTREAM_PROXY_PATH=../proxy/target/debug/radicle-proxy yarn run:dev:proxy",
```

Comment:
While we're here can we default `start` to be release? :pray: :innocent: :bow:

Reply:
Hm, I don't see a reason why not.

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/1489/commits/e4325829a97673b72553c87de86a78c676b62e37

Reply:
Rolling this back as @geigerzaehler pointed out this would slow down proxy recompilation when working on integrating proxy <-> UI.